test/extensions/clusters/aggregate/cluster_update_test.cc

```diff
@@ -170,22 +171,22 @@ TEST_F(AggregateClusterUpdateTest, LoadBalancingTest) {
 
   Upstream::HostConstSharedPtr host;
   for (int i = 0; i < 33; ++i) {
-    EXPECT_CALL(factory_.random_, random()).WillRepeatedly(Return(i));
+    EXPECT_CALL(random_, random()).WillRepeatedly(Return(i));
```

Comment:
it is not obvious to me why all these call-sites in this file and a similar one below had to change.

Reply:
They didn't, I just split my large refactor in the wrong direction.